notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb

```diff
@@ -140,140 +140,98 @@
-      "metadata": {
-        "id": "lWEdiXsJg0XY"
-      },
-      "source": [
-        "## Before you begin\n",
+        "import sys\n",
         "\n",
-        "### Set your project ID\n",
+        "if \"google.colab\" in sys.modules:\n",
         "\n",
-        "**If you don't know your project ID**, try the following:\n",
-        "- Run `gcloud config list`.\n",
-        "- Run `gcloud project list`.\n",
-        "- See the support page: [Locate the project ID](https://support.google.com/googleapi/answer/7014113)"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "oM1iC_MfAts1"
-      },
-      "outputs": [],
-      "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "    import IPython\n",
         "\n",
-        "# Set the project id\n",
-        "! gcloud config set project $PROJECT_ID"
+        "    app = IPython.Application.instance()\n",
+        "    app.kernel.do_shutdown(True)"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "region"
+        "id": "56e219dbcb9a"
       },
       "source": [
-        "#### Region\n",
-        "\n",
-        "You can also change the `REGION` variable, used by Vertex AI. Learn more about [Vertex AI regions](https://cloud.google.com/vertex-ai/docs/general/locations)"
+        "### Authenticate your notebook environment (Colab only)\n",
+        "Authenticate your environment on Google Colab."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "region"
+        "id": "c97be6a73155"
       },
       "outputs": [],
       "source": [
-        "REGION = \"us-central1\"  # @param {type: \"string\"}"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "dr--iN2kAylZ"
-      },
-      "source": [
-        "### Authenticate your Google Cloud account\n",
-        "Depending on your  Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below.\n",
+        "import sys\n",
+        "\n",
+        "if \"google.colab\" in sys.modules:\n",
         "\n",
-        "**1. Vertex AI Workbench**\n",
-        "- Do nothing as you are already authenticated.\n",
+        "    from google.colab import auth\n",
         "\n",
-        "**2. Local JupyterLab instance, uncomment and run:**"
+        "    auth.authenticate_user()"
       ]
     },
     {
-      "cell_type": "code",
-      "execution_count": null,
+      "cell_type": "markdown",
       "metadata": {
-        "id": "457c78b08293"
+        "id": "bf2dc7995ac2"
       },
-      "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "<div class=\"alert alert-block alert-warning\">,\n",
+        "<b>⚠️ The kernel is going to restart. Wait until it's finished before continuing to the next step. ⚠️</b>,\n",
+        "</div>"
```

Comment:
Please move this warning message to just under the Restart runtime code section (see notebook template for example)